change/@azure-msal-common-e4f59bc0-0846-4b3a-a6bd-b8a4829c5bff.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Implemented functionality to skip the cache for MI when claims are provided #7207",
```

Comment:
This is a good first PR, but it does not fully resolve the issue. The claims must be injected in leg1 and leg2 of FIC - this is separate work.